trace/sql.go

```diff
@@ -101,7 +101,7 @@ type (
 		// Safe replacement of context are provided only inside callback function
 		Context   *context.Context
 		Call      call
-		TxContext *context.Context
+		TxContext context.Context //nolint:containedctx
```

Comment:
Why you change the type from pointer to copy of context?

Reply:
For exclude case with replace TxContext from otel/opentracing adapter
Because it is not a user context, it is a context from start tx